llvm/lib/Object/BuildID.cpp

```diff
@@ -42,6 +43,21 @@ template <typename ELFT> BuildIDRef getBuildID(const ELFFile<ELFT> &Obj) {
   return {};
 }
 
+BuildIDRef getBuildID(const COFFObjectFile *Obj) {
```

Comment:
Could this (the ability to extract a build id from pe/coff) be split out and landed separately?

Reply:
By that, we don't have any test for it. The test `compiler-rt/test/profile/Windows/instrprof-binary-correlate.c` uses it.